ts/Series/Treegraph/TreegraphSeriesDefaults.ts

```diff
@@ -35,7 +35,7 @@ import { Palette } from '../../Core/Color/Palettes';
  *
  * TODO change back the demo path
  * @sample highcharts/demo/treegraph-chart
- *          Treegraph Chart
+ *         Treegraph Chart
```

Comment:
@pawellysy There is a todo about the missing sample. I do not see how this needs to be changed back. Can you take a look, please?

Reply:
We agreed with @TorsteinHonsi, that Treegraph won't be released until we change the design, so I changed all demos, to not be referenced in the official docs. More info here: #17600